docs/approvals/service-now/index.md

```diff
@@ -154,16 +182,28 @@ e.g `Octopus: Deploy "Web Site" version 1.0.1-hotfix-001 to "Dev"`
 The title must match the format **exactly**, including the double-quotes.
 :::
 
-## Known Issues and limitations
+### Respecting Change Windows
 
-- Pressing **TEST** on a ServiceNow connection that has not been saved will error. The workaround is to press **SAVE** and then open the connection flyout again and press **TEST**.
+:::warning
+This feature is only available for version 2022.3.3026 and later
+:::
 
-- The approval status of a deployment is not evaluated until the deployment scheduled start time has been reached.
+In addition to a change request being approved, a change must also be in its schedule change window in order for the deployment to execute.  The change window is controlled by the `Planned start date` and `Planned end date` on the linked change request.
 
-- The `Planned start date` and `Planned end date` supplied on a CR will not be considered.
+::: info
+The following list assumes the linked change is in an **approved** state.
+:::
 
-- Once a CR is deemed to be related to a deployment, then only this CR will be evaluated for the deployment to proceed. If the CR is incorrect, you will need to cancel the deployment, close the CR and try the deployment again.
+- If no `Planned start date` and `Planned end date` are specified there is no change window and the deployment will execute.
+- If only a `Planned start date` is set the deployment will execute on or after the defined date.
+- If only a `Planned end date` is set the deployment will execute on or before the defined date.
+- If `Planned start date` and `Planned end date` are specified the deployment will execute on or between the defined dates.
```

Comment:
This could be listed out as a table

              | No Start |  < Start | > Start
No End | Run now | Wait       | Run now         
  < End | Run now | Wait.      | Run now
 > End  | Fail.         | Fail.        | Fail

Or maybe not.